custom_components/zaptec/__init__.py

```diff
@@ -636,8 +654,8 @@ def cleanup_task(_task: asyncio.Task):
 
         self._trigger_task = self.config_entry.async_create_background_task(
             self.hass,
-            self._trigger_poll(obj),
-            f"Zaptec Poll Update for {obj.qual_id}",
+            self._trigger_poll(zaptec_obj),
+            f"Zaptec Poll Update for {zaptec_obj.qual_id}",
         )
         self._trigger_task.add_done_callback(cleanup_task)
```

Comment:
After this line was where I thought it would make sense to add `for coord in children_coordinators`.